Updating and Adding Wintercoats

Homingpenguins opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
This pr updates most of the wintercoat sprites and adds wintercoats to all jobs that didn't have it beforehand (besides prisioner)
The updated wintercoats are Hop, Hos, Cmo, Paramedic, Rd, Geneticist, and CE from the varients made earlier.
Ive added a wintercoat for chef, mime, clown, chaplain, bartender, curator, detective, lawyer, and psychologist.
I didnt make many of these sprites, so credit to each spriter is listed next to their respective coats, if it is empty, it is either mine or was kept from the last wintercoat pr. 

sprites ordered by department, head of staff on the left, followed by the wintercoats i kept from the previous ones, then any updated wintercoats, then entirely new wintercoats
![Wintercoat Showcase](https://user-images.githubusercontent.com/59744509/110258746-c6e32280-7f69-11eb-9497-e9442b63fe7f.PNG)
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
Gives the last remaining few jobs a wintercoat, and updates the head wintercoats with "better" sprites, if anyone thinks one of the old sprites are better, just let me know in the comments. It also vastly improves the organization of the sprites in the dmi, it was very disorganized and a pain to make look nice. the code is mostly the same, i did change a few things to make it look nicer, but thats about it.
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:
imageadd: Wintercoats for the service jobs without it, and the psychologist
imageadd: Updated a bunch of wintercoats that were a bit lacking, including all head of staff coats and the geneticist's coat.
imageadd: Credit for the sprites go to Jo☆, Fingus, Papaporo Paprito, Horatio22, Helianthus, and YouTubeBoy
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Homingpenguins committed:
adds, organizes, and updates

Homingpenguins committed:
should fix the integration test

i also removed the credit for the sprites in the code and will put credit in the cl.

Farquaar commented:
I feel like the clown coat should be more silly.

ATH1909 commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -644,22 +647,109 @@
+	armor = list("melee" = 5, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
+	hoodtype = /obj/item/clothing/head/hooded/winterhood/chap
+
+/obj/item/clothing/head/hooded/winterhood/chap
+	desc = "The Pope didnt forget to bless the most important part of the coat, thus giving off a holy feel."
+	icon_state = "winterhood_chap"
```
let this hold null rods and the other things that subtypes of /obj/item/clothing/suit/hooded/chaplainsuit can hold

ATH1909 commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -644,22 +647,109 @@
+	desc = "Warm, Stylish, approved by the space pope. Everything you need in a winter coat."
+	icon_state = "wintercoat_chap"
+	inhand_icon_state = "wintercoat_chap"
+	allowed = list(/obj/item/analyzer, /obj/item/stack/medical, /obj/item/dnainjector, /obj/item/reagent_containers/dropper, /obj/item/reagent_containers/syringe, /obj/item/reagent_containers/hypospray, /obj/item/healthanalyzer, /obj/item/flashlight/pen, /obj/item/reagent_containers/glass/bottle, /obj/item/reagent_containers/glass/beaker, /obj/item/reagent_containers/pill, /obj/item/storage/pill_bot ...
```
let this hold null rods and the other things that subtypes of /obj/item/clothing/suit/hooded/chaplainsuit can hold, not... whatever this copy+pasted list is.

ATH1909 commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -669,42 +759,40 @@
+	desc = "A random dectective decided to throw away the typical trenchcoat stariotype and make himself a wintercoat, this was the result."
+	icon_state = "wintercoat_det"
+	inhand_icon_state = "wintercoat_det"
+	armor = list("melee" = 35, "bullet" = 30, "laser" = 30,"energy" = 40, "bomb" = 25, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 50)
```
this should copy the stat's of the detective's trenchcoat, not his vest, or else it'll outclass both of them (the former in armor values, and the latter in area coverage)

ATH1909 commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -738,36 +840,35 @@
+	desc = "A simple white coat with a brown trim, perfect for its job. Just a tad bit resistant to chemicals."
+	icon_state = "wintercoat_psych"
+	inhand_icon_state = "wintercoat_psych"
+	armor = list("melee" = 5, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 10)
```
why does this get melee armor

ATH1909 commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -738,36 +840,35 @@
+/obj/item/clothing/head/hooded/winterhood/medical/psych
+	desc = "A brown hood that is also slightly resistant to chemicals, you should probably avoid them while wearing this anyways."
+	icon_state = "winterhood_psych"
+	armor = list("melee" = 5, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 10)
```
why does this get melee armor

ATH1909 commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -738,36 +840,35 @@
-	inhand_icon_state = "coatrd"
+	icon_state = "wintercoat_rd"
+	inhand_icon_state = "wintercoat_rd"
 	armor = list(MELEE = 0, BULLET = 0, LASER = 0,ENERGY = 0, BOMB = 20, BIO = 0, RAD = 0, FIRE = 30, ACID = 0)
```
... why do normal winter coats get 5 points of melee armor, but not some of these subtypes?

I mean, this PR didn't cause that weirdness, but maybe it could change/fix it. 🤷

Homingpenguins replied:
I tried to catch all instences of this weirdness left by the last pr, but this is one of them i missed, ill fix it.

ATH1909 commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -644,22 +647,109 @@
+	desc = "A black and white striped coat that the mimes made for themselves, who knows what this can do..."
+	icon_state = "wintercoat_mime"
+	inhand_icon_state = "wintercoat_mime"
+	armor = list("melee" = 20, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
```
"melee" = 20

um

Homingpenguins replied:
i dont even know how this one happened.

ATH1909 commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -669,42 +759,40 @@
-	icon_state = "winterhood_hos"
+/obj/item/clothing/suit/hooded/wintercoat/security/det
+	name = "\improper detective's winter coat"
+	desc = "A random dectective decided to throw away the typical trenchcoat stariotype and make himself a wintercoat, this was the result."
```
```suggestion
	desc = "One day, a particularly cold and bored detective decided to defy the typical trenchcoated detective stereotype and commission a winter coat for himself. The resulting winter coat design became quite popular among detectives stationed in cold environments."
```

ATH1909 commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -644,22 +647,109 @@
+
+/obj/item/clothing/suit/hooded/wintercoat/curator
+	name = "\improper curator's winter coat"
+	desc = "An acult looking wintercoat that the curator calls his own, he doesnt let anyone execpt himself touch it."
```
"acult"?????

*doesn't

*except

this also assumes the curator's gender

Homingpenguins replied:
the description has been completely rewritten

ATH1909 commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -644,22 +647,109 @@
+
+/obj/item/clothing/suit/hooded/wintercoat/clown
+	name = "\improper clown's winter coat"
+	desc = "A wintercoat made specially made for the clowns from an unknown place. All clowns have one, but not all of them wear it."
```
```suggestion
	desc = "A winter coat made specially made for the clowns from an unknown place. All clowns have one, but not all of them wear it."
```

ATH1909 commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -644,22 +647,109 @@
+	hoodtype = /obj/item/clothing/head/hooded/winterhood/mime
+
+/obj/item/clothing/head/hooded/winterhood/mime
+	desc = "A black and white hood that is slightly disorenting to look at, so you try not to look at it."
```
```suggestion
	desc = "A black and white hood that is slightly disorienting to look at, so you try not to look at it."
```

ATH1909 replied:
I'm gonna stop with the grammar edits here for now. You really ought to run this stuff through a spell-checker or something, though.

Homingpenguins committed:
I suck at discriptions...

Co-authored-by: ATH1909 <42606352+ATH1909@users.noreply.github.com>

Homingpenguins committed:
wintercoat changed to winter coat

all instences in the discription has also been fixed

Co-authored-by: ATH1909 <42606352+ATH1909@users.noreply.github.com>

Homingpenguins committed:
spelling

thanks ATH1909 for your editorial work, ill look over everything again.

Co-authored-by: ATH1909 <42606352+ATH1909@users.noreply.github.com>

Mothblocks commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -644,22 +647,109 @@
+	hoodtype = /obj/item/clothing/head/hooded/winterhood/chap
+
+/obj/item/clothing/head/hooded/winterhood/chap
+	desc = "The Pope didnt forget to bless the most important part of the coat, thus giving off a holy feel."
```
```suggestion
	desc = "The Pope didn't forget to bless the most important part of the coat, thus giving off a holy feel."
```

Mothblocks commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -644,22 +647,109 @@
+
+/obj/item/clothing/suit/hooded/wintercoat/chef
+	name = "\improper chef's winter coat"
+	desc = "A white winter coat that somehow isnt stained yet, maybe it will be one day."
```
```suggestion
	desc = "A white winter coat that somehow isn't stained yet, maybe it will be one day."
```

Mothblocks commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -644,22 +647,109 @@
+	icon_state = "wintercoat_chef"
+	inhand_icon_state = "wintercoat_chef"
+	allowed = list(/obj/item/reagent_containers/spray/plantbgone, /obj/item/plant_analyzer, /obj/item/seeds, /obj/item/reagent_containers/glass/bottle, /obj/item/reagent_containers/glass/beaker, /obj/item/cultivator, /obj/item/reagent_containers/spray/pestspray, /obj/item/hatchet, /obj/item/storage/bag/plants, /obj/item/graft, /obj/item/secateurs, /obj/item/geneshears)
+	armor = list("melee" = 5, "bullet" = 0, "laser" = 0,"energy" = 0, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 0, "acid" = 0)
```
This, and every other `armor` value, needs to use the defines (MELEE, BULLET, etc).

Homingpenguins replied:
working on this now.

Mothblocks commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -644,22 +647,109 @@
+	hoodtype = /obj/item/clothing/head/hooded/winterhood/clown
+
+/obj/item/clothing/head/hooded/winterhood/clown
+	desc = "A bright yellow and red hood that screams honk just by looking at it, tends to temporarily blind moths with just a glance."
```
```suggestion
	desc = "A bright yellow and red hood that screams 'honk' just by looking at it, tends to temporarily blind moths with just a glance."
```

Mothblocks commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -644,22 +647,109 @@
+
+/obj/item/clothing/suit/hooded/wintercoat/chap
+	name = "\improper chaplins's winter coat"
+	desc = "Warm, Stylish, approved by the space pope. Everything you need in a winter coat."
```
```suggestion
	desc = "Warm, stylish, approved by the space pope. Everything you need in a winter coat."
```

Mothblocks commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -644,22 +647,109 @@
+
+/obj/item/clothing/suit/hooded/wintercoat/curator
+	name = "\improper curator's winter coat"
+	desc = "One look at this coat and you would think the wearer is part of a cult, after looking at the owner, you relize that its probably true."
```
```suggestion
	desc = "One look at this coat and you would think the wearer is part of a cult. After looking at the owner, you realize that it's probably true."
```

Mothblocks commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -669,42 +759,40 @@
+	hoodtype = /obj/item/clothing/head/hooded/winterhood/security/det
+
+/obj/item/clothing/head/hooded/winterhood/security/det
+	desc = "The hood looks like it was taken straight from the security personell's coat, after a second glance, you see that its a slightly darker red."
```
```suggestion
	desc = "The hood looks like it was taken straight from the security personell's coat. After a second glance, you see that it's a slightly darker red."
```

ATH1909 replied:
*personnel's

Mothblocks commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -669,42 +759,40 @@
+	hoodtype = /obj/item/clothing/head/hooded/winterhood/security/law
+
+/obj/item/clothing/head/hooded/winterhood/security/law
+	desc = "Even though the coat isnt standered lawyer wear, the hood makes the entire outfit look decient for a lawyer to wear."
```
```suggestion
	desc = "Even though the coat isn't standard lawyer wear, the hood makes the entire outfit look decent for a lawyer to wear."
```

Twaticus reviewed (changes requested):
many problems, needs a big pass over. details for each dmi spoilered
all hoods/coats/inhands were compared against the base winterhood/wintercoat in their respective dmis

<details> 
  <summary> icons/mob/clothing/head.dmi </summary>

- 	winterhood_hop right/left side sprites have extra pixels on bottom
- 	winterhood_hydro right/left side sprites have extra pixels on bottom
- 	winterhood_chef right/left side sprites have extra pixels on bottom
- 	winterhood_chap back sprite missing shading
- 	winterhood_bar left side sprite offset
- 	winterhood_curate right/left side sprites have extra pixels on bottom
- 	winterhood_hos right/left side sprites have extra pixels on bottom & top
- 	winterhood_sec right/left side sprites have extra pixels on bottom & top
- 	winterhood_det right/left side sprites have extra pixels on bottom & top
- 	winterhood_cmo right/left side sprites have extra pixel on bottom & top, wrong color pixels
- 	winterhood_med right/left side sprites have extra pixel on bottom
- 	winterhood_chem right/left side sprites have extra pixel on bottom
- 	winterhood_viro right/left side sprites have extra pixel on bottom
- 	~~winterhood_para back sprite shade/outline backwards, left side sprite offset~~ **prefer previous paramed coat**
- 	winterhood_psych back sprite has extra pixels, right/left side sprites have extra pixel on bottom & top
- 	winterhood_rd right/left side sprites have extra pixel on bottom & top
- 	winterhood_sci right/left side sprites have extra pixel on bottom
- 	winterhood_gene back sprite has extra pixels, right/left side sprites have extra pixel on bottom & top, stray pixels, wrong color pixels, left side sprite offset
- 	winterhood_ce right/left side sprites have extra pixel on bottom & top
- 	winterhood_engie back sprite missing bottom row pixels
- 	winterhood_atmos right/left side sprites have extra pixel on bottom
- 	winterhood_qm right/left side sprites have extra pixel on bottom
- 	winterhood_cargo right/left side sprites have extra pixel on bottom
- 	winterhood_miner right/left side sprites have extra pixel on bottom

</details>

<details> 
  <summary> icons/mob/clothing/suit.dmi </summary>

- 	wintercoat_hop_t right side sprite has extra pixels on side
- 	wintercoat_jani front/back sprites have extra pixels on bottom
- 	wintercoat_jani_t front/back have extra pixels on bottom
- 	wintercoat_chef front sprite has extra pixels on top
- 	wintercoat_chef_t front sprite offset
- 	wintercoat_clown_t back sprite offset
- 	wintercoat_chap all sprites missing shading
- 	wintercoat_chap_t all sprites missing shading
- 	wintercoat_hos left side sprite has extra pixels on top
- 	wintercoat_chem front/back sprites have extra pixels on bottom
- 	wintercoat_chem_t front/back sprites have extra pixels on bottom
- 	wintercoat_viro front/back sprites have extra pixels on bottom, alpha pixels
- 	wintercoat_viro_t front/back sprites have extra pixels on bottom, alpha pixels
- 	wintercoat_psych front sprite offset, left/right side sprites are too wide + missing hand hole
- 	wintercoat_psych_t right side sprite has extra pixels on side, left/right side sprites missing hand hole
- 	wintercoat_rd duplicate sprites, left side sprite missing hand hole
- 	wintercoat_rd_t right side sprite has extra pixels on side
- 	wintercoat_sci alpha pixels
- 	wintercoat_sci_t alpha pixels
- 	wintercoat_gene back sprite has extra pixels on top, left/right side sprites are too wide
- 	wintercoat_gene_t right side sprite has extra pixels on side
- 	wintercoat_robo front/back sprites have extra pixels on bottom
- 	wintercoat_robo_t front/back sprites have extra pixels on bottom
- 	wintercoat_qm front/back sprites have extra pixels on bottom
- 	wintercoat_qm_t front/back sprites have extra pixels on bottom

</details>

<details> 
  <summary> icons/mob/inhands/clothing_lefthand.dmi </summary>

**all back sprites cover the mob, all right side sprites missing/messed up**
- 	wintercoat_chef front sprite offset, back sprite missing, wrong right side sprite, left side sprite offset
- 	wintercoat_mime wrong right side sprite, left side sprite offset
- 	wintercoat_clown wrong right side sprite, left side sprite offset
- 	wintercoat_chap wrong right side sprite, left side sprite offset
- 	wintercoat_bar wrong right side sprite, left side sprite offset
- 	wintercoat_curate wrong right side sprite, left side sprite offset
- 	wintercoat_det back sprite missing, wrong right side sprite, left side sprite offset
- 	wintercoat_law back sprite missing, wrong right side sprite, left side sprite offset
- 	~~wintercoat_para back sprite missing, wrong right side sprite, left side sprite offset~~ **prefer previous paramed coat**
- 	wintercoat_psych back sprite missing, wrong right side sprite, left side sprite offset
- 	wintercoat_gene front sprite offset, wrong right side sprite, left side sprite offset
- 	wintercoat_ce back sprite missing, wrong right side sprite

</details>

<details> 
  <summary> icons/mob/inhands/clothing_righthand.dmi </summary>

**all back sprites cover the mob, all left side sprites missing/messed up**

- 	wintercoat_chef back sprite missing, right side sprite offset, wrong left side sprite
- 	wintercoat_mime back sprite missing, right side sprite offset, wrong left side sprite
- 	wintercoat_clown back sprite missing, right side sprite offset, wrong left side sprite
- 	wintercoat_chap back sprite missing, right side sprite offset, wrong left side sprite
- 	wintercoat_bar back sprite missing, right side sprite offset, wrong left side sprite
- 	wintercoat_curate back sprite missing, right side sprite offset, wrong left side sprite
- 	wintercoat_det back sprite missing, right side sprite offset, wrong left side sprite
- 	wintercoat_law back sprite missing, right side sprite offset, wrong left side sprite
- 	~~wintercoat_para back sprite missing, right side sprite offset, wrong left side sprite~~ **prefer previous paramed coat**
- 	wintercoat_psych back sprite missing, right side sprite offset, wrong left side sprite
- 	wintercoat_gene back sprite missing, right side sprite offset, wrong left side sprite
- 	wintercoat_robo front sprite flipped, right side sprite flipped

</details>

<details> 
  <summary> items/obj/clothing/hats.dmi </summary>

- cmo_envirohelm/cmo_envirohelm-light change needs to be reverted

</details>

<details> 
  <summary> icons/obj/clothing/suits.dmi </summary>

- 	wintercoat_hop offset
- 	wintercoat_hop_t offset
- 	wintercoat_jani has extra pixels on bottom
- 	wintercoat_jani_t has extra pixels on bottom
- 	wintercoat_chef offset
- 	wintercoat_mime offset
- 	wintercoat_mime_t offset
- 	wintercoat_clown offset
- 	wintercoat_clown_t offset
- 	wintercoat_chap offset, missing shading
- 	wintercoat_chap_t offset, missing shading
- 	wintercoat_bar offset
- 	wintercoat_bar_t offset
- 	wintercoat_curate offset
- 	wintercoat_hos offset
- 	wintercoat_hos_t offset
- 	wintercoat_det offset
- 	wintercoat_det_t offset
- 	wintercoat_law offset
- 	wintercoat_law_t offset
- 	wintercoat_cmo offset
- 	wintercoat_cmo_t offset
- 	~~wintercoat_para offset, has directionals~~ **prefer previous paramed coat**
- 	~~wintercoat_para_t offset~~ **prefer previous paramed coat**
- 	wintercoat_psych offset, has directionals
- 	wintercoat_psych_t offset, has directionals
- 	wintercoat_rd offset
- 	wintercoat_rd_t offset
- 	wintercoat_gene offset
- 	wintercoat_gene offset
- 	wintercoat_robo has extra pixels on bottom
- 	wintercoat_robo_t has extra pixels on bottom
- 	wintercoat_ce offset
- 	wintercoat_ce_t offset
- 	wintercoat_qm has extra pixels on bottom
- 	wintercoat_qm_t has extra pixels on bottom

</details>

dont ask how long this took

Homingpenguins committed:
All the dmi changes

All the sprites should be consistent with the original. im doing a final sweep after the rest of the grammer changes.

Homingpenguins committed:
Update code/modules/clothing/suits/miscellaneous.dm

Co-authored-by: Mothblocks <35135081+Mothblocks@users.noreply.github.com>

Homingpenguins committed:
fixes the armor_list

Also forgot the paramed inhands and namings. should fix the broken integration test

Homingpenguins commented:
how do i tell the pr that the changes requested were actually changed?

Mothblocks commented:
The conversations resolve themselves.

Mothblocks reviewed (approved):
Code is good, up to Twaticus now.

Twaticus commented:
awesome thanks for going back over the sprites, just a couple more fixes

icons/mob/clothing/head.dmi: winterhood_gene right/left side sprites have stray pixels

icons/mob/inhands/clothing_lefthand.dmi: wintercoat_law front facing sprite is offset, wintercoat_gene front facing sprite is offset & right side sprite has a stray pixel

icons/mob/inhands/clothing_righthand.dmi: base wintercoat front facing sprite has a stray pixel, wintercoat_psych back facing sprite is offset

icons/obj/clothing/suits.dmi: wintercoat_curate is missing its backround, wintercoat_psych_t has unneeded directionals

Homingpenguins committed:
changes asked above.

Im doing the last sweep now, ive been super busy lately and havent been able to.

Homingpenguins committed:
fixes that mess.

once again, ive been busy and ive missed a bunch of stuff, all the conflicts are gone and i almost lost all my sprites :D.

Homingpenguins commented:
alright, ive fixed all the sprites, but i will go over it one last time in a little bit, so i wont open the pr quite yet.

optimumtact commented:
@Twaticus for sprite approval

Learner10 commented:
> 
> 
> I feel like the clown coat should be more silly.

The hoodie is just the top of a fuzzy banana.

Homingpenguins commented:
ive waited on guill's improvements on wintercoats to be merged, and now that they have, i finally have the time to work on this again.

Homingpenguins committed:
moves the sprites to the new wintercoat folder.

Homingpenguins committed:
more icon fixes

Fikou commented on code/modules/clothing/suits/wintercoats.dm:
```diff
@@ -17,7 +17,7 @@
 		/obj/item/toy,
 )
-	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 10, RAD = 0, FIRE = 0, ACID = 0)
+	armor = list(MELEE = 5, BULLET = 0, LASER = 0, ENERGY = 0, BOMB = 0, BIO = 10, RAD = 0, FIRE = 0, ACID = 0)
```
this random balance change isnt even on the winter hood counterpart and some other armor changes arent reflect on hoods as well, they should all be consistent between coat and hood

Sparkezel commented:
I liked old CE sprite more, it was more shiny.

GuillaumePrata commented:
Some suits have more available in the vendor than there are job slots for that role.
Also this PR is reverting some changes made on the winter coat files for clarity because you are using outdated files from before that Merge as a base and didn't update to it.

Homingpenguins commented:
im extremely busy with work right now, so ill come back to this pr later when the seasonal job calms down in about 2 weeks or so. I will get this finished sometime though

Mothblocks commented:
Let me know when you want it reopened.
